CONTRIBUTING.md

```diff
@@ -5,6 +5,10 @@ Code styles are predefined in the RubyMine/IntelliJ project files. Also, the
 [Rails Community Style Guide](https://github.com/bbatsov/ruby-style-guide) covers majority of the
 styles that we use.
 
+Our frontend code style follows Airbnb's [JavaScript](https://github.com/airbnb/javascript) and
+[React](https://github.com/airbnb/javascript/tree/master/react) style guide. Run `yarn lint` in the
```

Comment:
Actually original seems correct -> `Our frontend code style follows Airbnb's Javascript and React style guide`.